src/components/defectReport/DefectReportCreateForm.tsx

```diff
@@ -149,7 +149,7 @@ export const DefectReportCreateForm = ({
                 onChange,
                 name,
                 required: true,
-                label: texts.defectReport.categoryName,
+                label: texts.defectReport.categoryName + ' *',
```

Comment:
please see other lines in this file being formatted like this 
```
label={`${texts.defectReport.inputTitle} *`}
```

do you think it's fine to have the two different syntaxes? dann fell free to merge, otherwise you could change to an uniform format.

Reply:
I wrote the `placeholder` in the same format.

https://github.com/smart-village-solutions/smart-village-app-app/commit/537a83a3ec675e2563d4c93994bfdba2dce14873

Reply:
and why? what was you goal? the placeholder has no string concatenation or anything.

Reply:
So that all inputs are in the same format. But you're right. It doesn't need any extra logic.
I've reversed the change.

https://github.com/smart-village-solutions/smart-village-app-app/commit/d9a9f57fc10b44ca8700325a106c249dcd0ff009

Reply:
the same format should count for strings and values with concatenation, not for single variables